code/modules/mob/living/simple_animal/abnormality/zayin/bald.dm

```diff
@@ -25,37 +25,40 @@
 	gift_chance = 10
 	gift_message = "Now we're feeling awesome!"
 
-	var/list/balding_list = list(
-							"Bald"
-							)
+	var/list/balding_list = list("Bald")
```

Comment:
From what I can see, this is now redundant and unused.

Reply:
Will remove.